components/org.wso2.carbon.identity.branding.preference.management.core/src/main/java/org/wso2/carbon/identity/branding/preference/management/core/BrandingPreferenceManagerImpl.java

```diff
@@ -363,6 +365,25 @@ public void deleteCustomText(String type, String name, String screen, String loc
         }
     }
 
+    public void deleteAllCustomText() throws BrandingPreferenceMgtException {
```

Comment:
Missing method comment or add the annotation

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/20/commits/a5d063ddb596c1f82afbda520f8119eacc2cac6d